<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewController</class>
 <widget class="QMainWindow" name="ViewController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewController</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">    color: white;
                                        background-color: rgb(165, 29, 45);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;</string>
        </property>
        <property name="text">
         <string>Map Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMapName">
        <property name="styleSheet">
         <string notr="true">color: rgb(165, 29, 45);
                                        background-color: rgba(229, 165, 10, 0.95);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;
                                        border-color: rgb(165, 29, 45);
                                    </string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="backgroundList">
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(165, 29, 45);
color: white;
</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewMap"/>
    <addaction name="actionSaveMap"/>
    <addaction name="actionEditMap"/>
    <addaction name="ChangeBackground"/>
    <addaction name="ClearAll"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(229, 165, 10);

QPushButton {
    background-color: rgb(229, 165, 10);
    border-radius: 5px;
}

QPushButton:checked {
    background-color: rgb(165, 29, 45);
}</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFILE">
   <property name="text">
    <string>FILE</string>
   </property>
  </action>
  <action name="actionNewMap">
   <property name="text">
    <string>New Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveMap">
   <property name="text">
    <string>Save Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEditMap">
   <property name="text">
    <string>Edit Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="ClearAll">
   <property name="text">
    <string>ClearAll</string>
   </property>
   <property name="toolTip">
    <string>ClearAll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="ChangeBackground">
   <property name="text">
    <string>ChangeBackground</string>
   </property>
   <property name="toolTip">
    <string>ChangeBackground</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
